20_beta_diversity.Rmd

```diff
@@ -273,50 +273,66 @@ variable_names <- c("ClinicalStatus", "Gender", "Age")
 # Apply relative transform
 enterotype <- transformSamples(enterotype, method = "relabundance")
 
-# Get assay
-assay <- t(assay(enterotype, "relabundance"))
-# Get colData
-coldata <- colData(enterotype)
-
 # Create a formula
```

Comment:
Just a bit more explanation to support naive users might be useful here; like assay is the response variable, and the explanatory variables would be listed in variable_names but do not have to be named here yet.